backend.native/tests/interop/basics/bf.kt

```diff
@@ -53,10 +53,10 @@ fun test(s: S, x1: Long, x2: B2, x3: Short, x4: Int, x5: Int, x6: Long) {
 
     // Also check with some insignificant bits modified:
 
-    assign(s, x1 + 2, x2, (x3 + 8).toShort(), x4 - 16, x5 + 32, x6 + Long.MIN_VALUE)
+    assign(s, x1 + 2, x2, (x3 + 8u).toUShort(), x4 - 16u, x5 + 32, x6 + Long.MIN_VALUE)
     check(s, x1, x2, x3, x4, x5, x6)
 
-    assignReversed(s, x1 + 2, x2, (x3 + 8).toShort(), x4 - 16, x5 + 32, x6 + Long.MIN_VALUE)
+    assignReversed(s, x1 + 2, x2, (x3 + 8u).toUShort(), x4 - 16u, x5 + 32, x6 + Long.MIN_VALUE)
```

Comment:
Do we have unsigned in C ellipsis cases?

Reply:
What do you mean?

Reply:
Passing unsigned types to C `void foo(int, ...)`

Reply:
I have supported it with this PR.

Reply:
In tests.